### PR TITLE
feature: cmd queue, kill worker

### PR DESCRIPTION
**Pull request by ireznice**

### CODE REVIEW

File: lib/node_starter/cmd_consumer.rb [hunk 1]
```diff
@@ -0,0 +1,76 @@
+require 'bunny'
+
+module NodeStarter
+  # class wrapping connection to queue that stores uss nodes to be started
```
Comment (panjan) [inline lib/node_starter/cmd_consumer.rb]: huh?

File: lib/node_starter/cmd_consumer.rb [hunk 2]
```diff
@@ -0,0 +1,76 @@
...
+      queue_params = {
+        durable: false,
+        auto_delete: true,
+        exclusive: true
+      }
+
+      @queue = @channel.queue("", queue_params)
+    end
+
+    def register_id(id)
```
Comment (panjan) [inline lib/node_starter/cmd_consumer.rb]: register_node(id)?

File: lib/node_starter/cmd_consumer.rb [hunk 3]
```diff
@@ -0,0 +1,76 @@
+require 'bunny'
```
Comment (panjan) [inline lib/node_starter/cmd_consumer.rb]: specs missing

File: lib/node_starter/queue_subscribe.rb [hunk 4]
```diff
@@ -2,17 +2,20 @@
 
 require 'node_starter/starter'
 require 'node_starter/consumer'
+require 'node_starter/cmd_consumer'
 
 module NodeStarter
   # class for receiving start node messages
```
Comment (panjan) [inline lib/node_starter/queue_subscribe.rb]: and also stop

File: lib/node_starter/queue_subscribe.rb [hunk 5]
```diff
@@ -2,17 +2,20 @@
...
   class QueueSubscribe
     def initialize
       @consumer = NodeStarter::Consumer.new
+      @cmd_consumer = NodeStarter::CmdConsumer.new
```
Comment (panjan) [inline lib/node_starter/queue_subscribe.rb]: it's not clear to me what Consumer and CmdConsumer do

File: spec/integration/queue_subscribe_spec.rb [hunk 6]
```diff
@@ -18,13 +18,32 @@ def ack(_delivery_info)
   end
 end
 
+class FakeCmdConsumer
```
Comment (panjan) [inline spec/integration/queue_subscribe_spec.rb]: allow

**Comment by lksv** [comment]
hi @ireznice and @panjan, I already wrote to @h4rr0d that I expect current solutin with sidekiq is not ideal. Please discuss it with @h4rr0d.

**Pull request closed without merging**